terraform/azure/proxycommand.py

```diff
@@ -31,15 +31,13 @@ def delete_pod():
         [
             "kubectl",
             "run",
-            "--image",
-            POD_IMAGE,
+            f"--image={POD_IMAGE}",
             "--command",
             "--wait",
             POD_NAME,
             "--",
             "/bin/sh",
-            "-c",
-            SCRIPT,
+            f"-c={SCRIPT}",
```

Comment:
This will break. It depends on what CLI. I tried `sh -c=echo` and i got an error about it.

Reply:
Ok, will revert

Reply:
Done in [`c717bb3`](https://github.com/2i2c-org/infrastructure/pull/939/commits/c717bb3e943193ab2c1a8f9b5ba767b264bf7dbb)